src/cmd/linuxkit/run_hyperkit.go

```diff
@@ -116,6 +202,12 @@ func runHyperKit(args []string) {
 	h.Memory = *mem
 	h.DiskSize = *diskSz
 
+	// Add 9p and vsock for port forwarding if VPNKit is launched automatically
+	if *startVPNKit {
+		h.Sockets9P = []hyperkit.Socket9P{{Path: vpnKitPortSocket, Tag: "port"}}
+		h.VSockPorts = append(h.VSockPorts, 62373) // forwarding
```

Comment:
It's the vsock port used to forward network traffic from host->guest. I'll add a comment